src/content/en/tools/chrome-devtools/javascript/get-started.md

```diff
@@ -0,0 +1,171 @@
+   until DevTools highlights this line of code.
+
+When you checked the **click** checkbox, you set up an event-based breakpoint on
+all `click` events. When *any* node is clicked, and that node has a `click`
+handler, DevTools automatically pauses on the first line of the `click` handler.
+
+[resume]: /web/tools/chrome-devtools/images/resume-script-execution.png
+
+## Step 3: Step through the code
+
+Now that your code is paused, you can step through it, one line at a time, until
+you discover the line of code that is causing the bug.
+
+1. Click  **Step into next function call** ![Step into next function
+   call][into]{:.devtools-inline} to step through the execution of the
+   `onClick` function, one line at a time. DevTools highlights the
+   `if (inputsAreEmpty()) {` line of code. This highlight indicates the line of
+   code you're paused on.
+1. Press **Step into next function call** again. The `inputsAreEmpty` function
+   doesn't sound related to your bug, but the function isn't in this file, so
+   you can't know for sure.
+1. Take a look at the definition of `inputsAreEmpty`. This function most likely
+   isn't related to the bug, so click **Step out of current function** ![Step
+   out of current function][out]{:.devtools-inline}. DevTools
+   executes the rest of `inputsAreEmpty` before returning control to you.
+   DevTools is now highlighting the next line of `onClick`, which is
+   `updateLabel();`.
+1. Click **Step into next function call** again.
+   This function is probably related to the bug, so it's a good idea to step
+   through it, one line at a time. DevTools is now highlighting the first
+   line of the `updateLabel` function, which is `var addend1 = getNumber1()`.
+1. You *could* step into this function as well, since it's probably related to
+   the bug, but for the sake of learning, click **Step over next function
+   call** ![Step over next function call][over]{:.devtools-inline} instead.
+   DevTools executes the line of code to completion, without stepping into it.
+1. Look to the right of the line of code that you executed in the last step.
+   Notice how DevTools inserted `addend1 = "5"` to the right of the code. When
+   a variable is defined or changed, DevTools shows you the new value of the
+   variable. Look closely at the value of `addend1` that DevTools is showing.
+   The value `"5"` is surrounded in quotes,
+   which means that JavaScript is treating it as a string. This is the cause
+   of the bug. When `updateLabel` computes the sum, it concatenates the
+   values as strings, rather than adding them as numbers.
+
+[into]: /web/tools/chrome-devtools/images/step-into.png
+[out]: /web/tools/chrome-devtools/images/step-out.png
+[over]: /web/tools/chrome-devtools/images/step-over.png
+
+## Step 4: Apply a fix
+
+You've identified a potential cause for the bug. All that's left is to test out
+your fix by editing the code and then re-running the demo. You don't
```

Comment:
This sentence makes it sound like you're skipping the fix and going straight to the test. Then there's a fix in step 1. Can you please take another run at this transition.